### ISSUE TITLE
Free draw objects are using attributes from another file than other ER entities are using

### ISSUE THREAD

**Issue by a16tobfr** [HGustavs/LenaSYS#5837]
This was noticed when working on  a change request for #5675.
The free drawn objects are using an attribute from diagram_figure.js instead of diagram_symbol.js, which every other entity seem to use. 

This can make it very confusing and it is easy to make mistakes. May take an unnecessary amount of time to debug this type of error.

The attribute that was found were 'strokeColor' which exists in both diagram_symbol.js and diagram_figure.js.

Basically, this means that the free drawn objects are using attributes from both diagram_symbol.js and diagram_figure.js.

Would be good if the attribute that was used was the same as the attribute used by other entities (in diagram_symbol.js), if it is possible.

**Comment by a17vikab** [comment]
This Issue continues on issue: #5675 and therefore, it will be smart to wait and see if that issue gets approved for merge and later on continue working from that code.

**Comment by a17vikab** [comment]
Issue: #5675 merges the two files and there are two different kinds of objects and I want to argue that it is best to not merge the two functions that handle the two kinds of object. In conclusion, issue: #5837 got solved in #5675.

Another reason why it is best to leave the functions is that it will be huge refactoring and maybe not the highest priority at the moment.
